blocks/build/post-meta/index.css

```diff
@@ -0,0 +1 @@
+.wp-block-lsx-post-meta-basic input[type=text]{background:inherit;border:0;color:inherit;font-family:Gilbert,sans-serif;font-size:64px}
```

Comment:
It's preferable not to have any styles for basic elements in a block. Instead blocks should rely on the styles provided by the theme.

Reply:
Ive removed the styles, they are not needed for this block.  I will make sure to rely on the styles provided by the theme.
 - [commit](https://github.com/lightspeeddevelopment/lsx/commit/b59d0e5b6a8729fb03810f6466d20a5c9eab2b01)